python/jetToolbox_cff.py

```diff
@@ -22,7 +22,8 @@
 
 
 def jetToolbox( proc, jetType, jetSequence, outputFile,
-		updateCollection='', updateCollectionSubjets='',
+		updateCollection='', 
+		updateCollectionSubjets='', subjetSize = 0.4,
```

Comment:
naming issue: subjetSize should really be the fat jet size for rerunning the subjet b tagging.